GUI/MetaDraw.xaml.cs

```diff
@@ -250,31 +251,59 @@ private void DrawAnnotatedBaseSequence(MetaDrawPsm psm)
             }
 
             // draw b ions
-            foreach (var bIon in psm.FragmentIons.Where(p => p.ProductType == ProductType.B))
+            foreach (var bIon in psm.FragmentIons.Where(p => p.NeutralTheoreticalProduct.ProductType == ProductType.b))
```

Comment:
this old-style looping through the 4 accepted ion types seems out of date in the new era